pydantic_ai_slim/pydantic_ai/_output.py

```diff
@@ -98,15 +110,34 @@ def build(
         if output_type is str:
             return None
 
+        preferred_mode = None
         if isinstance(output_type, ToolOutput):
             # do we need to error on conflicts here? (DavidM): If this is internal maybe doesn't matter, if public, use overloads
             name = output_type.name
             description = output_type.description
             output_type_ = output_type.output_type
             strict = output_type.strict
+            preferred_mode = 'tool'
+        elif isinstance(output_type, StructuredOutput):
+            name = output_type.name  # TODO: Get this to the response_format model request arg
+            description = output_type.description  # TODO: Get this to the response_format model request arg
+            output_type_ = output_type.output_type
+            strict = output_type.strict  # TODO: Get this to the response_format model request arg
```

Comment:
I'm not understanding these TODOs, but if you do, then I guess that's good enough for now 😄